pkg/docker/docker.go

```diff
@@ -544,8 +545,11 @@ func runContainerTar(opts RunContainerOptions, imageMetadata *docker.Image) (cmd
 	return cmd, tarDestination
 }
 
-// this funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
-func runContainerDockerRun(container *docker.Container, d *stiDocker, image string) {
+// dumpContainerInfoAndWait dumps information about a running container (port/ip/etc) and then
+// waits for the container to exit (which it may never do).  Used when --run=true is passed
+// because the user wants to run the newly built image immediately.
+// This function runs forever, ensure a sig handler is installed elsewhere to handle cntrl-c.
```

Comment:
I am not... but per your other comment using "cntrl", I suspect this might
be something like the single x double spacing sentences?

https://en.wikipedia.org/wiki/CTRL

Rodolfo Carvalho

OpenShift Developer Experience

On Fri, Apr 1, 2016 at 10:05 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/docker/docker.go
> <https://github.com/openshift/source-to-image/pull/452#discussion_r58259996>
> :
>
> > @@ -544,8 +545,11 @@ func runContainerTar(opts RunContainerOptions, imageMetadata *docker.Image) (cmd
> >  	return cmd, tarDestination
> >  }
> >
> > -// this funtion simply abstracts out the running of the newly produced image when the --run=true option is provided
> > -func runContainerDockerRun(container *docker.Container, d *stiDocker, image string) {
> > +// dumpContainerInfoAndWait dumps information about a running container (port/ip/etc) and then
> > +// waits for the container to exit (which it may never do).  Used when --run=true is passed
> > +// because the user wants to run the newly built image immediately.
> > +// This function runs forever, ensure a sig handler is installed elsewhere to handle cntrl-c.
>
> you're just trolling me now, right?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/source-to-image/pull/452/files/64acc1bf3a18541084b4fb1c3ae969e419466e6d#r58259996>
>